scripts/update-browser-releases/index.ts

```diff
@@ -93,6 +101,23 @@ const options = {
     skippedReleases: [82], // 82 was skipped during COVID
     chromestatusURL: 'https://chromestatus.com/api/v0/channels',
   },
+  edge_desktop: {
+    bcdFile: './browsers/edge.json',
+    bcdBrowserName: 'edge',
+    browserEngine: 'Blink',
+    releaseBranch: 'Stable',
+    betaBranch: 'Beta',
+    nightlyBranch: 'Dev',
+    firstRelease: 12,
+    skippedReleases: [
+      12, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35,
+      36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53,
+      54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71,
+      72, 73, 74, 75, 76, 77, 78, 82,
+    ], // 82 was skipped during COVID
+    edgeupdatesURL:
+      'https://edgeupdates.microsoft.com/api/products?view=enterprise',
```

Comment:
Just discovered that you can remove the `?view=enterprise` param. I also discovered that you can access specific channels with `/api/prodcuts/stable`, `/api/products/beta`, etc. (although you probably want to get the data in one go).

Reply:
Fixed.